ajax/libs/shards-vue/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "shards-vue",
+  "filename": "dist/shards-vue.common.min.js",
```

```suggestion
  "filename": "shards-vue.common.min.js",
```

As auto-update is configured with `dist` as the basePath, the files within will be available at the root of the package in cdnjs.